src/h_vialib/client.py

```diff
@@ -29,14 +31,15 @@ def is_pdf(self):
 class ViaClient:  # pylint: disable=too-few-public-methods
     """A small wrapper to make calling Via easier."""
 
-    def __init__(self, service_url, host_url):
+    def __init__(self, service_url, host_url, secret_token):
         """Initialize a ViaClient pointing to a `via_url` via server.
 
         ￼
         :param via_url location of the via server
         :param host_url origin of the request
         """
         self.service_url = urlparse(service_url)
+        self._secure_url = ViaSecureURL(secret_token)
```

Comment:
We refer to this as just `secret` everywhere else. I'm not sure if saying token would imply it's more than a random string?